packages/auth/src/AuthService.ts

```diff
@@ -0,0 +1,44 @@
+import Keycloak from "keycloak-js";
+
+export class AuthService{
+    static auth: any = {};
+
+    loadUserProfile(): any {
+        // Retrieve User Profile
+        return new Promise((resolve, reject) => {
+            AuthService.auth.authz.loadUserProfile().success((profile: any) => {
```

Comment:
This is not going to work. What we probably should od is:
` return AuthService.auth.authz.loadUserProfile()` and remove all bellow